commands.js

```diff
@@ -130,6 +130,7 @@ var commands = exports.commands = {
 			switch (targetCmd) {
 			case 'me':
 			case 'announce':
+				break;
 			case 'invite':
 				var targetRoomid = toId(target.substr(8));
```

Comment:
From my earlier pullreq #1404, which is now redundant because yours does the same thing but adds extra stuff:

    var targetRoomid = toId(target.substr(targetCmdIndex + 1));

This future proofs the code in case we ever create new aliases for `/invite`.